### PR TITLE
Mention minimum Ember version

### PR DESCRIPTION
**Pull request by machty**
I believe it's Ember 4, but if it's not I can change it, just so long as it's mentioned somewhere.

_edit: Ember 3.24+ seems more correct, and i verified this with my own testing_

### CODE REVIEW

**Comment by kpfefferle** [comment]
Hmm, we're definitely using Glint with 3.28... 🤔

**Comment by machty** [comment]
@kpfefferle I updated it to mention 3.24 as the minimum version after I confirmed with some testing of my own.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ TypeScript-powered tooling for Glimmer templates.
 
 ## Overview
 
-[Glint] is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
+[Glint] is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] (Version 3.24+) and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
```
Comment (chriskrycho) [inline README.md:7]:
```suggestion
[Glint] is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] v3.24+ and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
```

File: docs/ember/installation.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-To use Glint with [Ember](https://github.com/emberjs/ember.js), you'll add the `@glint/core` and `@glint/environment-ember-loose` packages to your project's `devDependencies`, then add a `"glint"` key to your project's `tsconfig.json`.
+To use Glint with [Ember](https://github.com/emberjs/ember.js) (Version 3.24 or higher), you'll add the `@glint/core` and `@glint/environment-ember-loose` packages to your project's `devDependencies`, then add a `"glint"` key to your project's `tsconfig.json`.
```
Comment (chriskrycho) [inline docs/ember/installation.md:1]:
```suggestion
To use Glint with [Ember](https://github.com/emberjs/ember.js) v3.24 or higher, you'll add the `@glint/core` and `@glint/environment-ember-loose` packages to your project's `devDependencies`, then add a `"glint"` key to your project's `tsconfig.json`.
```

File: docs/overview.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Glint is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
+Glint is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] (Version 3.24+) and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
```
Comment (chriskrycho) [inline docs/overview.md:1]:
```suggestion
Glint is a set of tools to aid in developing code that uses the Glimmer VM for rendering, such as [Ember.js] v3.24+ and [GlimmerX] projects. Similar to [Vetur] for Vue projects or [Svelte Language Tools], Glint consists of a CLI and a language server to provide feedback and enforce correctness both locally during editing and project-wide in CI.
```

**Review by chriskrycho — CHANGES_REQUESTED** [review]
Thanks! Our projects (not just Typed Ember but Ember) generally use `v` instead of `Version`, so I suggested that change. Once that's done, we'll merge it.

**Comment by machty** [comment]
@chriskrycho accepted your changes

**Review by chriskrycho — APPROVED** [review]

**Comment by chriskrycho** [comment]
Windows failure is spurious. Another timeout issue, @dfreeman.

**Pull request merged**